Common/Models/WatchPredictedGlucose.swift

```diff
@@ -16,7 +16,7 @@ struct WatchPredictedGlucose: Equatable {
     let values: [PredictedGlucoseValue]
 
     init?(values: [PredictedGlucoseValue]) {
-        guard values.count > 1 else {
+        guard values.count > 2 else {
```

Comment:
`guard values.count > 1` should ensure that there are 2 or more values, which would allow accesses of values[0] and values[1].

Changing to `guard values.count > 2` means that three or more values are required, which is more than needed, I believe.

Reply:
right... not sure what I was thinking :)

Reply:
Well, the crash really points to something like this.  I looked at the stack trace, and reviewed the code; it does seem like an array access would cause that exact crash, but I can't see how we can end up with < 2 values.

Reply:
turns out it is related to casting Double to Int16 when the double is out of the Int16 bounds.